Source/WTF/wtf/CagedPtr.h

```diff
@@ -50,36 +48,31 @@ class CagedPtr {
 
     CagedPtr() : CagedPtr(nullptr) { }
     CagedPtr(std::nullptr_t)
-        : m_ptr(shouldTag ? tagArrayPtr<T>(nullptr, 0) : nullptr)
+        : m_ptr(nullptr)
     { }
 
-    CagedPtr(T* ptr, size_t size)
-        : m_ptr(shouldTag ? tagArrayPtr(ptr, size) : ptr)
+    CagedPtr(T* ptr, size_t)
+        : m_ptr(ptr)
     { }
```

Comment:
Can this constructor be removed completely?  I suspect it's unused, or we can remove its use.

Reply:
It is used, but `size_t` part can be removed.